### PR TITLE
fix: TT-168 Add the parameters startDate and endDate in the path

### PR DESCRIPTION
**Pull request by VanessaIniguez**
Add the parameters startDate and endDate in the path used by the findEntriesByProjectId method of the EntryService service.

### CODE REVIEW

**Comment by kellycastrof** [comment]
Yes, but locally the tests were successful.

**Review by Angeluz-07 — APPROVED** [review]

File: src/app/modules/time-clock/services/entry.service.ts [hunk 1]
```diff
@@ -77,4 +80,14 @@ export class EntryService {
...
   }
+
+  getDateLastMonth() {
+    return (moment().subtract(1, 'months')).format();
+  }
+
+  getCurrentDate() {
+    return moment().format();
+  }
+
```
Comment (scastillo-jp) [inline src/app/modules/time-clock/services/entry.service.ts:91]: remove one of the black line

File: src/app/modules/time-clock/services/entry.service.spec.ts [hunk 2]
```diff
@@ -143,10 +143,13 @@ describe('EntryService', () => {
 
   it('entries are found by project id with a limit 2 by default', () => {
     const projectId = 'project-id';
+    const startDate = (moment().subtract(1, 'months')).format();
```
Comment (scastillo-jp) [inline src/app/modules/time-clock/services/entry.service.spec.ts:146]: I have a question.
What is the default format in this `format()` part?
I think you can add a specific format:  example .format('MM/DD/YYYY');  .format('MMM')

Comment (VanessaIniguez) [reply src/app/modules/time-clock/services/entry.service.spec.ts:146]: The format used to send start_date and end_date in the path is 2021-03-10T15:42:13-05:00,  at the moment I think it should be sent like this

**Review by scastillo-jp — APPROVED** [review]

**Review by thegreatyamori — APPROVED** [review]

**Review by kellycastrof — APPROVED** [review]

**Review by LEON12699 — APPROVED** [review]

**Pull request merged**